frmts/rdb/rdbdataset.cpp

```diff
@@ -19,6 +19,8 @@
 #include <cmath>
 #include <sstream>
 
+#include "cpl_float.h"
+
```

Comment:
let's revert that to avoid conflicts, as this driver is going to be removed anyway

```suggestion
```

Reply:
Done.